### PR TITLE
Bugfix/arch 619

### PR DESCRIPTION
**Pull request by shafi1379**
@rk-fsdev @afoerster  Changes made ony in ui/containers/WorkspaceDetails/sagas.ts, constants.ts and index.ts

### CODE REVIEW

**Comment by rk-fsdev** [comment]
@shafi1379  Travis build is failing now, pls take a look at it, and fix building errors.

File: common/src/main/scala/io/phdata/services/EmailService.scala [hunk 1]
```diff
@@ -58,15 +58,15 @@ class EmailServiceImpl[F[_]: Effect](context: AppContext[F], workspaceService: W
           values + ("userName" -> s"${user.getOrElse(LDAPUser("Unknown", "Unknown", DistinguishedName("cn=Unknown"), Seq.empty, None)).name}")
         )
       )
-      addressList = context.appConfig.approvers.notificationEmail
-      fromAddress = context.appConfig.smtp.fromEmail
```
Comment (martin-morek) [inline common/src/main/scala/io/phdata/services/EmailService.scala]: Why those two lines were removed? They are needed.

File: common/src/main/scala/io/phdata/services/EmailService.scala [hunk 2]
```diff
@@ -58,15 +58,15 @@ class EmailServiceImpl[F[_]: Effect](context: AppContext[F], workspaceService: W
...
     } yield {
       addressList.map(
         recipient => context.emailClient.send("A New Workspace Is Waiting", email, fromAddress, recipient)
       )
     }
   }
 
+
+
```
Comment (martin-morek) [inline common/src/main/scala/io/phdata/services/EmailService.scala]: please run code formation before each commit (`sbt scalafmt`)

**Comment by afoerster** [comment]
@shafi1379 is this tested? @rk-fsdev look good to you?

**Review by rk-fsdev — APPROVED** [review]

**Pull request merged**